WordPress/Classes/Utility/Universal Links/Route.swift

```diff
@@ -58,3 +75,56 @@ extension Route {
         return path == route.path
     }
 }
+
+// MARK: - Tracking
+
+/// Where did the deep link originate?
+///
+enum DeepLinkSource: Equatable {
+    case link
+    case banner
+    case email(campaign: String)
+    case widget
+    case inApp(presenter: UIViewController?)
+
+    init?(string: String) {
```

Comment:
Question: would it have any value if instead of returning a nil value, we add an `case` for this (e.g. `unknown`)?

Reply:
I think currently it's probably okay as-is, because we're only really using it in the widget situation right now.